ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/rlpx/handshake/ecies/ResponderHandshakeMessageV1.java

```diff
@@ -31,7 +31,7 @@
  *
  * <h3>Message structure</h3>
  *
- * The following describes the message structure:
+ * <p>The following describes the message structure:
```

Comment:
Seems unrelated, but ok.

Reply:
Yea, some cleanups slipped in.  There will be a bigger Java 17 pr later.

Reply:
undone.